src/components/Titlebar.vue

```diff
@@ -84,6 +87,38 @@ export default class Titlebar extends Vue {
 
     download(FILENAME, JSON.stringify(editorsSaved));
   }
+
+  private newProject() {
+    // TODO FE-55 Implement better Confirm dialog
+    if (window.confirm('Are you sure you want to create a new project? '
+      + 'All unsaved progress will be lost.')) {
+      const blankProject = {
+        overviewEditor: {
+          name: 'Overview',
+          editorState: {
+            nodes: [],
+            connections: [],
+          },
+        },
+        modelEditors: [{
+          name: 'untitled',
+          editorState: {
+            nodes: [],
+            connections: [],
+            panning: {
+              x: 0,
+              y: 0,
+            },
+            scaling: 1,
+          },
+        }],
+        dataEditors: [],
+        trainEditors: [],
+      };
+      this.loadEditors(blankProject);
```

Comment:
I don't think `TitleBar` should know what a blank project is. It should instead call a muation that resets the all the necessary fields. Load editors should not be used either since that is intended for loading already created editors, which is why you had to create an editor state on line 105. What happens if the editor state representation changes? We would have to change the code here as well which is not ideal.

What I suggest is to call a mutation that literally resets the vuex state to the exact same way it was when loading the page for the first time. This can be seen in `store/editors/index.ts`:
```
  currEditorType: EditorType.MODEL,
  currEditorIndex: 0,
  editorNames: new Set<string>(['untitled']),
  overviewEditor: {
    name: 'Overview',
    editor: newEditor(EditorType.OVERVIEW), // TODO: Lazy create?
  },
  modelEditors: [
    {
      name: 'untitled',
      editor: newEditor(EditorType.MODEL),
    },
  ],
  dataEditors: [],
  trainEditors: [],
```

Reply:
My proposed approach may not trigger a re-render if the name of the current editor is `untitled` when you load a blank project. This is because the key of `<baklava-editor>` is the editor name. 

I am going to implement UUIDs so this shouldn't be a problem in the future so if you experience this problem then I suggest doing what I have done in `loadEditors` mutation in the mean time:
```
 // TODO: Remove this hack - causes <baklava-editor> to re-render
    state.modelEditors[0].name += ' ';
```

Reply:
You're right, I was not too confident in doing this but for some reason I couldn't just override the state and trigger the rerender. It might have been because of duplicate keys that you fixed with UUIDs! I've now created a `resetState` mutation.